src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.RoatpOversight.Domain
+{
+    public static class GatewayReviewStatus
+    {
+        public const string Pass = "Pass";
+    }
```

Comment:
Can you update this class to be
public class GatewayReviewStatus
    {
        public const string Fail = "Fail";
        public const string Pass = "Pass";
    }